### ISSUE TITLE
Pathfinding layer generation error

### ISSUE THREAD

**Issue by aliaksei135** [aliaksei135/seedpod_ground_risk#62]
When adding a path layer on top of a risk layer, generation results in a following error:
``` python
Traceback (most recent call last):
  File "seedpod_ground_risk\core\plot_server.py", line 213, in compose_overlay_plot
  File "<__array_function__ internals>", line 5, in flipud
  File "numpy\lib\twodim_base.py", line 148, in flipud
ValueError: Input must be >= 1-d.
Input must be >= 1-d.
```

Looks like this isn't anything to do with layer itself, but the plot server not being able to handle it.

**Comment by Zach10a** [comment]
I've been tracking this down and it appears that the issue comes from line 213 in plot_server.py. raster_grid appears not to be producing anything in some situations.

**Comment by aliaksei135** [comment]
The usual cause for that is an OSM query failing and it returning a null. I'll close this as a duplicate of #61 in that case and we can reopen if it keeps occurring after #61 is fixed.
